contracts/ownership/Ownable.sol

```diff
@@ -19,7 +19,6 @@ contract Ownable {
    */
   constructor() public {
     _owner = msg.sender;
-    emit OwnershipTransferred(address(0), _owner);
```

Comment:
My bad for not being clear, I meant we should skip the test, not emitting the event :stuck_out_tongue:

Reply:
okay :smile: , no problem. Will add that.